web/packages/plugins/shared/common/src/components/NewEntityButton.tsx

```diff
@@ -52,7 +52,7 @@ export function NewEntityButton(props: {
             justifyContent: 'space-between',
           }}
         >
-          <Label label={'Folder for new entity'} />
+          <Label label={typeToCreate} />
```

Comment:
Why is this changed? The DestinationPicker should have a description on what you are picking for